### PR TITLE
Add yaml configuration examples to the "Custom Payment Gateway" cookbook

### PR DESCRIPTION
**Pull request by DennisCodeBuds**
In my projects I tend to use YAML for the configuration files so I thought it might be helpful to others to add configuration examples in the YAML format

| Q               | A
| --------------- | -----
| Branch?         | 1.9, 1.10 or master 
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | 
| License         | MIT

### CODE REVIEW

**Comment by CoderMaggie** [comment]
The base of this pull-request was changed, you need fetch and reset your local branch 
if you want to add new commits to this pull request. **Reset before you pull, else commits
may become messed-up.**

Unless you added new commits (to this branch) locally that you did not push yet, 
execute `git fetch origin && git reset "patch-1"` to update your local branch.
 
Feel free to ask for assistance when you get stuck :+1:

File: docs/cookbook/payments/custom-payment-gateway.rst [hunk 1]
```diff
@@ -41,7 +41,7 @@ In the following example, a new gateway will be configured, which will send paym
             }
         }
 
-    And at the end of ``src/Resources/config/services.xml`` add such a configuration for your gateway:
+    And at the end of ``src/Resources/config/services.xml``, or, ``src/Resources/config/services.yaml`` add such a configuration for your gateway:
```
Comment (arti0090) [inline docs/cookbook/payments/custom-payment-gateway.rst:44]: Such a comma spam 😨 
```suggestion
    And at the end of ``src/Resources/config/services.xml`` or ``src/Resources/config/services.yaml`` add such a configuration for your gateway:
```

File: docs/cookbook/payments/custom-payment-gateway.rst [hunk 2]
```diff
@@ -89,6 +100,15 @@ In the following example, a new gateway will be configured, which will send paym
             <tag name="sylius.gateway_configuration_type" type="sylius_payment" label="Sylius Payment" />
             <tag name="form.type" />
         </service>
+    
+    .. code-block:: yml
```
Comment (arti0090) [inline docs/cookbook/payments/custom-payment-gateway.rst:104]: As in other files we are using. Also other codeblocks have same issue
```suggestion
    .. code-block:: yaml
```

File: docs/cookbook/payments/custom-payment-gateway.rst [hunk 3]
```diff
@@ -89,6 +100,15 @@ In the following example, a new gateway will be configured, which will send paym
...
+    
+        #src/Resources/config/services.yaml
```
Comment (arti0090) [inline docs/cookbook/payments/custom-payment-gateway.rst:106]: Same in other codeblocks
```suggestion
        # src/Resources/config/services.yaml
```

**Review by CoderMaggie — APPROVED** [review]

**Comment by CoderMaggie** [comment]
Thanks, @DennisCodeBuds! :tada:

**Pull request merged**